### PR TITLE
SAI Release Notes for 1.6.3

### PR DESCRIPTION
**Pull request by pyuvarajan**
SAI Release Notes for 1.6.3

### CODE REVIEW

File: doc/SAI_1.6.3_ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+11. saiqueue
+12. sairouterinterface
+13. saiswitch
+14. saitunnel
+15. saitypes
+
+
+### sah.h		
+
+saimacsec and saisystem port API has bee added
```
Comment (ragu-raman) [inline doc/SAI_1.6.3_ReleaseNotes.md:122]: Typo error "been"

File: doc/SAI_1.6.3_ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+
+### saiacl.h	
+
+Provides generalized Access Control Lists management functions. The ACL contains three types of objects, ACL table, ACL entry and ACL counter. The ACL table contains a number of ACL entries. Each ACL table defines a set of unique matching fields for all its ACL entries. A packet can match rules in different ACL tables and take non-conflicting actions from all the matched rules. However, within an ACL table, if a packet matches multiple rules, only the actions from the rule of highest priority are executed. ACL counters can also be created and attached to an ACL entry in order to counter the number of packets or bytes that match the ACL entry.
```
Comment (ragu-raman) [inline doc/SAI_1.6.3_ReleaseNotes.md:126]: Typo - count the number of packets

**Comment by kannankvs** [comment]
SONiC 202006 is using SAI 1.6.3 and hence we shall use only the 1.6.3 release notes and ignore the 1.6.5 release notes. We have addressed all the review comments given in the [PR1134](https://github.com/opencomputeproject/SAI/pull/1134) meant for 1.6.5 in this 1.6.3 release notes. Request you to review this PR and sign off.

**Comment by kannankvs** [comment]
****Review Comments from PR1134 applicable for SAI 1.6.3**** 
@bandaru-viswanath 
>>(1) I think the saitunnel.h has more diffs, than what is presented here.
<Reply> We have added more details. Request you to kindly check and confirm.

@michaelli10 
>>(1) Does it make sense to separate the changes by feature similar to SAI 1.5 release notes for context? Also, adding the PR references would be helpful. https://github.com/opencomputeproject/SAI/blob/v1.6/doc/SAI_1.5_ReleaseNotes.md
<Reply> Yes, We have revamped the SAI 1.6.3 release notes. We also added the list of features mentioned by @rlhui in this rel notes. Request you to kindly check and confirm.

>> (2) Is the SAI_LAG_ATTR_TPID attribute intentionally included?
<Reply> Yes, as it is included in the new SAI 1.6.3 release tag.

>> (3) Will you add the system port changes (and other saiport.h changes) from [PR#1080](https://github.com/opencomputeproject/SAI/pull/1080) [PR#1081](https://github.com/opencomputeproject/SAI/pull/1081) ?
<Reply> Yes, the revamped document covers the new APIs and its additional changes which are part of [PR#1081](https://github.com/opencomputeproject/SAI/pull/1081) , however [PR#1080](https://github.com/opencomputeproject/SAI/pull/1080) is not part of this SAI 1.6.3 release tag. Kindly suggest if we need to add the [PR#1080](https://github.com/opencomputeproject/SAI/pull/1080) as well part of this release notes.

>> (4) Can you add an item about the new system port related attributes?
SAI_SWITCH_ATTR_SWITCH_ID
SAI_SWITCH_ATTR_MAX_SYSTEM_CORES
SAI_SWITCH_ATTR_SYSTEM_PORT_CONFIG_LIST
SAI_SWITCH_ATTR_NUMBER_OF_SYSTEM_PORTS
SAI_SWITCH_ATTR_SYSTEM_PORT_LIST
SAI_SWITCH_ATTR_NUMBER_OF_FABRIC_PORTS
SAI_SWITCH_ATTR_FABRIC_PORT_LIST

<Reply> Yes, we have added details on this. Request to kindly check and confirm.


****NOTE:** The other comments related rephrasing, typo are also addressed.** 
The comments that are given in [PR#1134](https://github.com/opencomputeproject/SAI/pull/1134) are specific to SAI 1.6.5, but not relevant to SAI 1.6.3 are ignored.


@phshaikh , @bandaru-viswanath , @marian-pritsak , @AshokDaparthi , @JaiOCP ,@DeanKalla, @michaelli10, @rlhui: Request you all to review this and sign off.

**Review by rlhui — APPROVED** [review]

**Pull request merged**